config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3937,9 +3937,28 @@
 # 2) Firefox users are not provided, as they have not yet implemented
 #    Content-Security-Policy: sandbox.
 # DEFAULT: true, unless inlinehtml or separateDomains is on
-# lessonbuilder.use-csp-headers=false # 
+# OPTIONAL: This can take a string of which specific CSP headers to use, in this case it also defaults to true (Unless overrideen by separate content domain)
+# The default set of headers if not specified is 
+# sandbox allow-forms allow-scripts allow-top-navigation allow-popups allow-pointer-lock allow-popups-to-escape-sandbox
+# There are currently over 40 directives and growing. https://content-security-policy.com/
+# See that reference for the full list
+#
+# lessonbuilder.use-csp-headers=false 
+
+
+# OPTIONAL: This can take a string of which specific CSP headers to use, in this case it also defaults to true (Unless overrideen by separate content domain)
+# The default set of headers if not specified is 
+# sandbox allow-forms allow-scripts allow-top-navigation allow-popups allow-pointer-lock allow-popups-to-escape-sandbox
+
+# lessonbuilder.use-csp-headers=false
```

Comment:
I think this section above is duplicated?

Reply:
Yeah, looks like it was just maybe cherry-picked in twice.